File: dimik/dimik-32.py
repeat = int(input())
for x in range(repeat):
    innum, limiter = input().split()
    innum, limiter = int(innum), int(limiter)
    if limiter < innum:
        print("Invalid!")
    for x in range(1, int(limiter / innum) + 1):
        print(x * innum)
    print()

File: dimik/dimik-32.cpp
#include <iostream>
using namespace std;
int main()
{
    int repeat, i, j, x, y;
    cin >> repeat;
    for(i=1;i<=repeat;i++)
    {
        cin >> x >> y;
        if (x>y)
        {
            cout << "Invalid!" <<endl;
        }
        else
        {
            for(j=0; j+x <= y; j = j+x)
        {
            cout << j+x <<endl;
        }
        }
     cout<<endl;
    }
    return 0;
}
